assets/js/main.js

```diff
@@ -1,4 +1,17 @@
 wp.domReady(() => {
+	const copyBtn = document.querySelector('.ep-copy-button');
+	const successMessage = document.querySelector('.ep-copy-button-wrapper__success');
+	const request = copyBtn.getAttribute('data-request');
+
+	copyBtn.addEventListener('click', function () {
+		navigator.clipboard.writeText(request).then(function () {
```

Comment:
Getting this error `Uncaught TypeError: Cannot read properties of undefined (reading 'writeText')`

Reply:
@burhandodhy I wasn't able to reproduce this one. In which browser are you seeing this?